Library/Formula/duck.rb

```diff
@@ -0,0 +1,29 @@
+class Duck < Formula
+  homepage "https://duck.sh/"
+  url "https://dist.duck.sh/duck-src-16336.tar.gz"
+  version "4.6.2.16336"
```

Comment:
We can easily add the marketing version to the source tarball filename such as `duck-src-4.6.2.16336.tar.gz ` if that is what you suggest.